test/wallet.spec.ts

```diff
@@ -0,0 +1,152 @@
+import { BrowserContext } from 'playwright-core';
+import { Dappwright } from '../src';
+import { clickOnLogo, openProfileDropdown } from '../src/wallets/metamask/actions/helpers';
+import launchBrowser from './helpers/launchBrowser';
+
+describe('when interacting with the wallet', () => {
+  let browserContext: BrowserContext, wallet: Dappwright, _;
+
+  beforeAll(async () => {
+    [browserContext, _, wallet] = await launchBrowser();
+  });
+
+  afterAll(async () => {
+    await browserContext.close();
+  });
+
+  it('should lock and unlock', async () => {
+    await wallet.lock();
+    await wallet.unlock('password1234');
+  });
+
+  describe('when adding a network', () => {
+    afterAll(async () => {
+      await wallet.switchNetwork('local');
```

Comment:
is `local` a valid network for it to switch to?

Reply:
no, but part of the test is to validate partial matches